tests/simulator/test_run.py

```
@@ -194,6 +197,9 @@ def plot(diag_dir):
             vmin=-2,
             vmax=2,
         )
+        ru.GetPressure(time).plot(
```

Ruff (0.11.9)

Undefined name `ru`

(F821)